packages/central-server/src/apiV2/entityTypes/GetEntityTypes.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { allowNoPermissions } from '../../permissions';
+import { GETHandler } from '../GETHandler';
+
+/**
+ * Custom implementation required for this route as there is no corresponding DatabaseModel for EntityType
+ * (it's an enum not a table)
+ */
+export class GetEntityTypes extends GETHandler {
+  async assertUserHasAccess() {
+    await this.assertPermissions(allowNoPermissions);
+  }
+
+  async getEntityTypes() {
+    return this.models.entity.getEntityTypes();
+  }
+
+  async getDbQueryOptions() {
+    const { limit, page } = this.getPaginationParameters();
+    const offset = limit * page;
+
+    return { limit, offset };
+  }
+
+  async findSingleRecord(recordId) {
+    const entityTypes = await this.getEntityTypes();
+    const entityType = entityTypes.find(type => type === recordId);
+    if (!entityType) {
+      return undefined;
+    }
+
+    return { id: entityType, type: entityType };
+  }
+
+  async findRecords(criteria, options) {
```

Comment:
Chose to not support some of the basic behaviour (filtering, sorting, etc.) since the code around it is all so database heavy. However, chose to still support pagination as that isn't easy to disable in the Admin Panel